README.md

```diff
@@ -149,9 +159,9 @@ To run tests, run the following command:
 
 ## 🔭 Future Features <a name="future-features"></a>.
 
-- [ ] **[Share recipes]**
-- [ ] **[Like recipes]**
-- [ ] **[Add images for the recipes]**
+- [ ] **[express more features]**
+- [ ] **[Add transactions]**
+- [ ] **[View complex transactions]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] As mentioned by the previous reviewer, as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=requires%20that%20you-,give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.,-Interactions), at the `Acknowledgements` section please give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

Reply:
@Omar-Muhamad  fixed and implemented